lib/usecases/get_cached_surveys_use_case.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_survey/database/survey_storage.dart';
+import 'package:flutter_survey/model/survey_model.dart';
+import 'package:flutter_survey/usecases/base/base_use_case.dart';
+import 'package:injectable/injectable.dart';
+
+@Injectable()
+class GetCachedSurveysUseCase extends SimpleUseCase<List<SurveyModel>> {
```

Comment:
Do we have unit test for this new usecase?

Reply:
Woops, I missed this, will add 🙏